HW10/read.txt

```diff
@@ -0,0 +1,39 @@
+var b = [5, 10, 23, 10, 7, 9, 39, 16, 1, 12, 36, 20, 50];
+
+
+ 
+ console.log (b);
+ var i=0;
+ var y=[];
+ 
+ for(i=0;i<b.length;i++){
+      
+ 	y[i]=(b[i]*10-5);
+ 	console.log (y[i]);
+ 	y[i]=y[i]+b[i];
```

Comment:
@NoorAlkattan on line 13 you are changing the value on array y, while the requirements was just print